packages/core/src/modules/connections/services/ConnectionService.ts

```diff
@@ -163,20 +163,28 @@ export class ConnectionService {
     config?: {
       myLabel?: string
       myImageUrl?: string
+      autoAcceptConnection?: boolean
     }
   ): Promise<ConnectionProtocolMsgReturnType<ConnectionRequestMessage>> {
     const connectionRecord = await this.connectionRepository.getById(connectionId)
 
     connectionRecord.assertState(ConnectionState.Invited)
     connectionRecord.assertRole(ConnectionRole.Invitee)
 
+    const { myLabel, myImageUrl, autoAcceptConnection } = config ?? {}
```

Comment:
Nit: I think it looks cleaner to do the default empty method assignment in the method parameters

Reply:
Good tip 👍 Let's try that